src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -78,9 +82,15 @@ class VisitService(
     return VisitDto(visitEntity)
   }
 
-  @Transactional(readOnly = true)
+  @Deprecated("See find visits pageable", ReplaceWith("findVisitsByFilterPageable(visitFilter).content"))
   fun findVisitsByFilter(visitFilter: VisitFilter): List<VisitDto> {
-    return visitRepository.findAll(VisitSpecification(visitFilter)).sortedBy { it.visitStart }.map { VisitDto(it) }
+    return findVisitsByFilterPageable(visitFilter).content
```

Comment:
Would also change the sort order here by name!? just encase they don't do paging